components/script/script_thread.rs

```diff
@@ -1840,6 +1840,9 @@ impl ScriptThread {
             WebDriverScriptCommand::AddCookie(params, reply) => {
                 webdriver_handlers::handle_add_cookie(&*documents, pipeline_id, params, reply)
             },
+            WebDriverScriptCommand::DeleteCookies(reply) => {
+                webdriver_handlers::handle_delete_cookies(&*documents,reply)
```

Comment:
We should pass pipeline_id in order to choose a particular document.

Reply:
Okay I'll delete cookies of a particular document.
But I think this [discussion](https://github.com/cypress-io/cypress/issues/408) is worth a read.